apps/hashdotdev/src/_pages/blog/0014_structured-data-value.mdx

```diff
@@ -0,0 +1,129 @@
+
+### Structure and simulation
+
+The role of structured data in all of this is quite simple.
+
+**Predictive simulations of real-world things are extremely easy to get wrong.** Many interesting systems that users typically want to simulate end up being "complex systems" in some respect, demonstrating some degree of nonlinearity due to emergent or adaptive behaviors. This is both because (a) any system involving a human is inherently complex, and because (b) "closed systems" (the only ones we can fully model without doubt, where all parameters and environmental variables are known) quite simply do not exist in the real world. Simulations are simplifications of the real world: downsized, abstracted, or analogical.
+
+**Complex systems are typically extremely sensitive to their initial conditions.** Ensuring an "as close to accurate" initial simulation starting point is therefore key. Pre-structured data, especially if tested and reviewed, or produced in accordance with a well-understood process or pipeline, can improve confidence in this. The more volatile a simulation target is, the greater the degree to which realtime or timely data availability matters in turn.
+
+**Structured data can be used for verification and validation, in addition to improving the accuracy of a simulation's initial state.** Structured data can be used to backtest simulation experiments, as well as to test the accuracy and correctness of individual behaviors themselves. In a world where behaviors and whole simulation models may be generated by foundation models, having a concrete picture of reality to compare to provides humans with an improved sense of a model's likely correctness.
+
+### More for humans than machines
+
+All this serves to highlight an interesting possibility. If to date structured data has existed for the benefit of _machines_, is it possible that in the future its primary use will in fact be to us, _humans_, in verifying and validating the outputs of generative AI models: simulation, language, multimodal, or other?
+
+Inherently tied to the idea of structured data is the idea of **typing** information. Data may be of a certain 'type' if it follows a set of rules established by that type.
+
+The [Block Protocol type system](https://blockprotocol.org/docs/types) outlines three different kinds of types which can help us think about structured data: entity types, property types, and data types.
+
+1.  **Entities** (semantic 'things' in a database) have entity types.
+
+- **Entity types** describe the _properties_ and type(s) of any entity, including both special-cased _link entities_ and ordinary, plain old _standard entities_.
+- A **link entity** corresponds to the relationship, or 'thing' that connects other entities. For example, a standard entity `Person` may be connected by a link entity called `Spouse` to another standard entity `Person`.
```

Comment:
Or maybe 'Married To'? I think `Spouse` is a bit more confusing because it refers to a party to the relationship rather than the relationship itself
```suggestion
- A **link entity** corresponds to the relationship, or 'thing' that connects other entities. For example, a standard entity `Person` may be connected by a link entity called `Marriage` to another standard entity `Person`.
```

Reply:
```suggestion
- A **link entity** corresponds to the relationship, or 'thing' that connects other entities. For example, two standard `Person` entities may be connected by a `Married To` link entity.
```